src/main/java/uk/ac/cam/cl/dtg/segue/dao/content/ContentMapper.java

```diff
@@ -353,20 +357,20 @@ public ObjectMapper generateNewPreconfiguredContentMapper() {
         ChoiceDeserializer choiceDeserializer = new ChoiceDeserializer(contentDeserializer, itemDeserializer);
 
         QuestionValidationResponseDeserializer validationResponseDeserializer 
-            = new QuestionValidationResponseDeserializer(
-                contentDeserializer, choiceDeserializer);
+            = new QuestionValidationResponseDeserializer(contentDeserializer, choiceDeserializer);
 
-        IsaacQuestionBaseDeserializer isaacQuestionBaseDeserializer = new IsaacQuestionBaseDeserializer(contentDeserializer);
+        IsaacQuestionBaseDeserializer isaacQuestionBaseDeserializer =
+                new IsaacQuestionBaseDeserializer(contentDeserializer);
 
         SimpleModule contentDeserializerModule = new SimpleModule("ContentDeserializerModule");
         contentDeserializerModule.addDeserializer(ContentBase.class, contentDeserializer);
         contentDeserializerModule.addDeserializer(IsaacQuestionBase.class, isaacQuestionBaseDeserializer);
         contentDeserializerModule.addDeserializer(Choice.class, choiceDeserializer);
         contentDeserializerModule.addDeserializer(Item.class, itemDeserializer);
         contentDeserializerModule.addDeserializer(QuestionValidationResponse.class, validationResponseDeserializer);
+        contentDeserializerModule.addDeserializer(
+                LLMMarkingExpression.class, new LLMMarkingExpressionDeserializer(objectMapper));
```

Comment:
Why does this need its own `ObjectMapper`? The other deserialisers all just use the jsonParser provided as first argument to the method calls:
```java
ObjectMapper mapper = (ObjectMapper) jsonParser.getCodec();
```